rllib/env/multi_agent_env_runner.py

```diff
@@ -181,6 +191,11 @@ def _sample_timesteps(
             self._episode = self._new_episode()
             self._make_on_episode_callback("on_episode_created")
 
+            # Erase all cached ongoing episodes (these will never be completed and
+            # would thus never be returned/cleaned by `get_metrics` and cause a memory
+            # leak).
+            self._ongoing_episodes_for_metrics.clear()
```

Comment:
Could be seen on long cloud runs I did on the weekend: 156 GB eaten away :) (but we have to be fair: there was alos a problem with the `_QueueActor` and `WandBLoggingActor`).